dspace-server-webapp/src/main/java/org/dspace/app/rest/ClarinDiscoJuiceFeedsDownloadService.java

```diff
@@ -269,7 +269,7 @@ private static JSONArray downloadJSON(String url) {
             Object obj = parser.parse(new InputStreamReader(conn.getInputStream()));
             return (JSONArray) obj;
         } catch (IOException | ParseException e) {
-            log.error("Failed to obtain/parse " + url + "\nCheck timeouts, redirects, shibboleth config.\n" + e);
+            log.debug("ERROR: Failed to obtain/parse " + url + "\nCheck timeouts, redirects, shibboleth config.\n" + e);
```

Comment:
I know why we do not log this as error. We want to reduce logs. But there is already reduction in cron job from each 2 minutes to each 2 hours. In prod environment, isn't this still an error?

Reply:
Yeah I forgot that, thank you